mit_tab_bot/roles.py

```diff
@@ -0,0 +1,117 @@
+import discord
+
+from .api import (
+    is_judge,
+    is_debater,
+    get_judge,
+    get_debater
+)
+
+ADMIN = [
+    'Talos#5956'
```

Comment:
Can you explain what exactly this corresponds to? Is this your personal user id or something?

Reply:
Yeah this is my personal Discord ID which grants superuser, plan is to make this part of the form on the deployer, and passed through an ENV variable, but I haven't finished the deployer piece yet so here it is.

Reply:
Gotcha, yeah would be nice to have this set via some env var/config file and then referenced by a constant like `ADMIN_DISCORD_ID` to make things clear